etc/input.conf

```diff
@@ -118,7 +118,7 @@
 #Alt+0 set current-window-scale 0.5     # halve the window size
 #Alt+1 set current-window-scale 1.0     # reset the window size
 #Alt+2 set current-window-scale 2.0     # double the window size
-#d cycle deinterlace                    # toggle the deinterlacing filter
+#d cycle-values deinterlace "yes" "no"  # toggle the deinterlacing filter
```

Comment:
why not cycle back to auto?

Reply:
The actual answer is that I was too lazy to update more documentation last night.